pandas-stubs/__init__.pyi

```diff
@@ -52,7 +87,7 @@ def read_sql(
 def read_feather(p: Union[Path, IO]) -> DataFrame: ...
 def to_numeric(
     arg: Union[int, float, List, Tuple, _np.ndarray, Series],
-    errors: Literal['ignore', 'raise', 'coerce'] = ...,
-    downcast: Literal['integer', 'signed', 'unsigned', 'float'] = ...,
+    errors: Literal["ignore", "raise", "coerce"] = ...,
+    downcast: Literal["integer", "signed", "unsigned", "float"] = ...,
```

Comment:
it's weird that this shows up, because I already made this change on master. I'd guess your branch is not completely up-to-date.

Reply:
Indeed I was behind master.